UI/window-youtube-actions.cpp

```diff
@@ -594,6 +607,95 @@ void OBSYoutubeActions::UiToBroadcast(BroadcastDescription &broadcast)
 	// Current time by default.
 	broadcast.schedul_date_time = QDateTime::currentDateTimeUtc().toString(
 		SchedulDateAndTimeFormat);
+
+	SaveSettings(broadcast);
+}
+
+void OBSYoutubeActions::SaveSettings(BroadcastDescription &broadcast)
+{
+	OBSBasic *main = OBSBasic::Get();
+
+	config_set_string(main->basicConfig, "YouTube", "Title",
+			  QT_TO_UTF8(broadcast.title));
+	config_set_string(main->basicConfig, "YouTube", "Description",
+			  QT_TO_UTF8(broadcast.description));
+	config_set_string(main->basicConfig, "YouTube", "Privacy",
+			  QT_TO_UTF8(broadcast.privacy));
+	config_set_string(main->basicConfig, "YouTube", "CategoryID",
+			  QT_TO_UTF8(broadcast.category.id));
+	config_set_bool(main->basicConfig, "YouTube", "MadeForKids",
+			broadcast.made_for_kids);
+	config_set_string(main->basicConfig, "YouTube", "Latency",
+			  QT_TO_UTF8(broadcast.latency));
+	config_set_bool(main->basicConfig, "YouTube", "AutoStart",
+			broadcast.auto_start);
+	config_set_bool(main->basicConfig, "YouTube", "AutoStop",
+			broadcast.auto_start);
+	config_set_bool(main->basicConfig, "YouTube", "DVR", broadcast.dvr);
+	config_set_bool(main->basicConfig, "YouTube", "ScheduleForLater",
+			broadcast.schedul_for_later);
+	config_set_string(main->basicConfig, "YouTube", "Projection",
+			  QT_TO_UTF8(broadcast.projection));
+	config_set_bool(main->basicConfig, "YouTube", "SettingsSaved", true);
+}
+
+void OBSYoutubeActions::LoadSettings()
+{
+	OBSBasic *main = OBSBasic::Get();
+
+	const char *title =
+		config_get_string(main->basicConfig, "YouTube", "Title");
+	ui->title->setText(QT_UTF8(title));
+
+	const char *desc =
+		config_get_string(main->basicConfig, "YouTube", "Description");
+	ui->description->setPlainText(QT_UTF8(desc));
+
+	const char *priv =
+		config_get_string(main->basicConfig, "YouTube", "Privacy");
+	int index = ui->privacyBox->findData(priv);
+	ui->privacyBox->setCurrentIndex(index);
+
+	const char *catID =
+		config_get_string(main->basicConfig, "YouTube", "CategoryID");
+	index = ui->categoryBox->findData(catID);
+	ui->categoryBox->setCurrentIndex(index);
+
+	bool forKids =
+		config_get_bool(main->basicConfig, "YouTube", "MadeForKids");
+	if (forKids)
+		ui->yesMakeForKids->setChecked(true);
+	else
+		ui->notMakeForKids->setChecked(true);
```

Comment:
I believe YouTube require you to manually specify the Made For Kids setting for every broadcast, even if you're reusing settings.

Reply:
Depends on your settings I think.
I have my channel set to not made for kids by default.
For neither the auto created stream nor scheduled streams I have to specify this manually.
But for scheduled streams I get asked to review the settings when reusing them.

Reply:
I've removed the auto setting of this option for now.